app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -105,6 +106,44 @@ describe("FormAudience", () => {
     );
   });
 
+  it("renders with disabled locale/country for applications that don't support them", async () => {
+    render(
+      <Subject
+        experiment={{
+          ...MOCK_EXPERIMENT,
+          application: NimbusExperimentApplication.DESKTOP,
+        }}
+        config={{
+          ...MOCK_CONFIG,
+          applicationConfigs: [
+            {
+              application: NimbusExperimentApplication.DESKTOP,
+              channels: [{ label: "Nightly", value: "NIGHTLY" }],
+              supportsLocaleCountry: false,
+            },
+          ],
+        }}
+      />,
+    );
+    await screen.findByTestId("FormAudience");
+    expect(
+      screen.getByTestId("locales").querySelector("input"),
+    ).toHaveAttribute("disabled");
+    expect(
+      screen.getByTestId("countries").querySelector("input"),
+    ).toHaveAttribute("disabled");
+    expect(
+      screen.getByText(
+        "This application does not currently support targeting by locale.",
+      ),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        "This application does not currently support targeting by country.",
+      ),
+    ).toBeInTheDocument();
```

Comment:
Another fun tip, just bc I'm here: you can replace this whole expect with `await screen.findByText("the text")`, it has the same effect as `expect(...).toBeInTheDocument()`